remake.yml

```diff
@@ -15,21 +15,31 @@ targets:
   all:
     depends: 3_visualize/out/figure_1.png
 
-  site_data:
-    command: download_nwis_data()
+  # site_data:
+  #   command: download_nwis_data()
+
+  # Download data
+  1_fetch/out/nwis_01427207.csv:
+    command: download_nwis_site_data(target_name)
+  
+  1_fetch/out/nwis_01432160.csv:
+    command: download_nwis_site_data(target_name)
   
+  1_fetch/out/nwis_01436690.csv:
+    command: download_nwis_site_data(target_name)
+  
+  1_fetch/out/nwis_01466500.csv:
+    command: download_nwis_site_data(target_name)
+    
+  site_data:
+    command: combine_site_data('1_fetch/out/nwis_01427207.csv', '1_fetch/out/nwis_01432160.csv', '1_fetch/out/nwis_01436690.csv', '1_fetch/out/nwis_01466500.csv')
```

Comment:
small suggestion, but you could write this as
```yaml
  site_data:
    command: combine_site_data(
      '1_fetch/out/nwis_01427207.csv', 
      '1_fetch/out/nwis_01432160.csv', 
      '1_fetch/out/nwis_01436690.csv', 
      '1_fetch/out/nwis_01466500.csv')
```
Is not necessary here, but if you had another site or two, it would get a bit long. Main thing I wanted to share here is that the YAML format does give you some options that may help with readability in these cases.

Reply:
This is very helpful! Wasn't sure if YAML would be weird about a line break.

Thanks for the timely review